src/node_crypto.cc

```diff
@@ -3443,14 +3450,22 @@ void CipherBase::InitIv(const FunctionCallbackInfo<Value>& args) {
 
   THROW_AND_RETURN_IF_NOT_STRING(args[0], "Cipher type");
   THROW_AND_RETURN_IF_NOT_BUFFER(args[1], "Key");
-  THROW_AND_RETURN_IF_NOT_BUFFER(args[2], "IV");
+
+  if (!Buffer::HasInstance(args[2]) && !args[2]->IsNull()) {
+    return env->ThrowTypeError("IV must be a buffer or null");
+  }
```

Comment:
Is this about `createCipheriv`? Why would any sane person come up with the idea to use a function `createCipheriv` for ciphers where there is no IV?